api/editor/api-editor/src/main/java/ca/bc/gov/tno/core/excel/CBRAReport.java

```diff
@@ -483,58 +520,63 @@ private int addPercentOfTotalRunningTime(int startRowIndex) {
     var sources = newsRadio.stream().collect(Collectors.groupingBy(Content::getSource));
     var sourceKeys = new ArrayList<String>(sources.keySet());
 
-    for (var i = 0; i < maxRows; i++) {
-      var row = sheet.createRow(++startRowIndex);
-
-      if (i < sources.size()) {
-        var sourceKey = sourceKeys.get(i);
-        var sourceRows = sources.get(sourceKey);
-
-        var name = sourceKey != null ? sourceKey : "NOT SET";
-        cellA = row.createCell(0);
-        cellA.setCellStyle(style);
-        cellA.setCellValue(name);
+    var valueStyle = createStyle((short) 12, false, true, HorizontalAlignment.CENTER, green);
+    valueStyle.setDataFormat(workbook.createDataFormat().getFormat("0.00%"));
 
-        var totalSource = sourceRows.stream().filter(c -> c.getFileReferences().size() > 0)
-            .mapToInt(c -> c.getFileReferences().get(0).getRunningTime()).sum();
-        var time = calculateRunningTimePercent(totalSource, totalRunningTime);
-        cellB = row.createCell(1);
-        cellB.setCellStyle(style);
-        cellB.setCellValue(time);
-      }
+    for (var i = 0; i < sources.size(); i++) {
+      var row = sheet.createRow(++startRowIndex);
+      var sourceKey = sourceKeys.get(i);
+      var sourceRows = sources.get(sourceKey);
+
+      var name = sourceKey != null ? sourceKey : "NOT SET";
```

Comment:
We should never have `null` values for these in the database with good data, but I have to handle the scenario.  Currently I'm labelling them as **NOT SET**